web/cobrands/northamptonshire/base.scss

```diff
@@ -11,10 +11,9 @@
 @import "../sass/mixins";
 @import "../sass/base";
 
+// Northamptonshire have asked for the logos to be removed from the header,
 #site-logo {
-    width: 190px;
-    background: url(/cobrands/northamptonshire/img/NorthAndWestLogo.png) 0 50% no-repeat;
-    background-size: contain;
+    background: none;
```

Comment:
@chrismytton This is displaying on top of the navigation, sorry for not spotting - think it needs to be display:none?

Reply:
Oh dear, how did I miss that?! Will change…

Reply:
Fixed in https://github.com/mysociety/fixmystreet/commit/dc1eda83412203f8b5ee41249090fab6f6f77205. Thanks @dracos 👍